app/javascript/modules/listingDetailsLottery/ListingDetailsLotterySearchForm.tsx

```diff
@@ -36,7 +36,7 @@ export const ListingDetailsLotterySearchForm = ({
   )
 
   const onSubmit = (data: { lotterySearchNumber: string }) => {
-    const { lotterySearchNumber } = data
+    const lotterySearchNumber = data?.lotterySearchNumber
```

Comment:
If the `data` param is not guaranteed to have `lotterySearchNumber`, the function should be typed to indicate that.

Reply:
Agreed!